aspnet/tutorials/publish-to-azure-webapp-using-vs.rst

```diff
@@ -1,47 +1,181 @@
 
-Start by either creating a new ASP.NET web app or opening an existing ASP.NET web app. 
+- Install `.NET Core + Visual Studio tooling <http://go.microsoft.com/fwlink/?LinkID=798306>`__
 
-1. In **Solution Explorer** of Visual Studio, right-click on the project and select **Publish**.
+- Verify your `Azure account <https://portal.azure.com/>`__. You can `open a free Azure account <https://azure.microsoft.com/pricing/free-trial/>`__ or `Activate Visual Studio subscriber benefits <https://azure.microsoft.com/pricing/member-offers/msdn-benefits-details/>`__.
 
-.. image:: publish-to-azure-webapp-using-vs/_static/01-Publish.png
+Create a web app
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-2. In the **Publish Web** dialog box, click on **Microsoft Azure Web Apps** and log into your Azure subscription.
+From the Visual Studio **Start** page, tap **New Project**.
 
-.. image:: publish-to-azure-webapp-using-vs/_static/02-PublishWebdb.png
+.. image:: first-mvc-app/start-mvc/_static/new_project.png
 
-3. Click **New** in the **Select Existing Web App** dialog box to create a new Web app in Azure.
+Alternatively, you can use the menus to create a new project. Tap **File > New > Project**.
 
-.. image:: publish-to-azure-webapp-using-vs/_static/03-SelectExistingWebAppdb.png
+.. image:: first-mvc-app/start-mvc/_static/new_project.png
 
-4. Enter a site name and region. You can optionally create a new database server, however if you've created a database server in the past, use that. When you're ready to continue, click **Create**.
+Complete the **New Project** dialog:
 
-.. image:: publish-to-azure-webapp-using-vs/_static/04-CreateWebAppOnMicrosoftAzuredb.png
+- In the left pane, tap **Web**
+- In the center pane, tap **ASP.NET Core Web Application**
+- Tap **OK**
 
-Database servers are a precious resource. For test and development it's best to use an existing server. There is **no** validation on the database password, so if you enter an incorrect value, you won't get an error until your web app attempts to access the database.
+.. image:: publish-to-azure-webapp-using-vs/_static/new_prj.png
 
-5. On the **Connection** tab of the **Publish Web** dialog box, click **Publish**.
+In the **New ASP.NET Core Web Application (.NET Core)** dialog:
 
-.. image:: publish-to-azure-webapp-using-vs/_static/05-PublishWebdb.png
+- Tap **Web Application**
+- Verify **Authentication** is set to **Individual User Accounts**
+- Verify **Host in the cloud** is **not** checked
+- Tap **OK**
 
-You can view the publishing progress in the **Web Publish Activity** window within Visual Studio.
+.. image:: publish-to-azure-webapp-using-vs/_static/noath.png
 
-.. image:: publish-to-azure-webapp-using-vs/_static/06-WebPublishActivityWindow.png
+Test the app locally
+^^^^^^^^^^^^^^^^^^^^^
 
-When publishing to Azure is complete, your web app will be displayed in a browser running on Azure. 
+- Use **Ctl-F5** to run the app in non-debug mode
```

Comment:
I would remove "in non-debug mode" because when you run the app we apply the Development hosting:environment